src/main/scala/org/bblfsh/client/cli/ScalaClientCLI.scala

```diff
@@ -1,6 +1,7 @@
 package org.bblfsh.client.cli
 
-import org.bblfsh.client.BblfshClient
+import org.bblfsh.client.v2.BblfshClient
+import org.bblfsh.client.v2.BblfshClient._
```

Comment:
It's needed in order for implicits defined in there to work, so `resp.uast.decode` becomes a thing.

Reply:
Can you please help me understand, this method attaches to any `ByteBuffer` as long as the import is enabled, right? Won't it collide with `decode` defined by some other package?

Reply:
>  this method attaches to any ByteBuffer as long as the import is enabled, right
right

>  Won't it collide with decode defined by some other package?

if some other package adds the same implicit, that gets imported here as well?
It will, with compile-time error.

Reply:
Resolving, as presume this answers your question @dennwc . But please, feel free to un-resolve it if it's not the case.

Reply:
I'm okay with it as long as it's a common practice in Scala :)